### PR TITLE
Rph chat integrated

### PR DESCRIPTION
**Pull request by RPHaynes**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -254,9 +255,27 @@ function App() {
...
+			.catch((x) => {
+				setToken("");
+				localStorage.setItem("token", "");
+			});
+	}
+
 	const [activeTheme, updateActiveTheme] = useState(0);
 	return (
 		<ThemeProvider theme={themes[activeTheme].theme}>
+			{/* Resnders Chat Box */}
```
Comment (LutherNa) [inline src/App.js]:
```suggestion
			{/* Renders Chat Box */}
```

File: src/Components/Client.js [hunk 2]
```diff
@@ -1,85 +1,88 @@
...
+		return;
+	}, []);
 	const actions = [
 		{ icon: <AddBoxIcon />, name: "Join/Create Chat Room" },
 		{ icon: <IndeterminateCheckBoxIcon />, name: "Leave Chat Room" },
 	];
-	var chatrooms = [];
-	var chatroomObject = { socket: null, username: null, room: null };
-
+	var [chatrooms, updateChatrooms] = useState([]);
```
Comment (LutherNa) [inline src/Components/Client.js]: I was under the impression that all useState ~~elements~~ ~~variables~~ _things_ are supposed to be declared as constant. 
```suggestion
	const [chatrooms, updateChatrooms] = useState([]);
```

**Review by LutherNa — APPROVED** [review]
Approved, but please look at the suggestions before merging.

**Review by LutherNa — APPROVED** [review]
I was all set to re-approved, but then I remembered I don't need to because new commits don't obliviate an old approval. Which is actually weird, I don't like that, you could commit anything and still use my approval. 
Regardless, Client.js could probably use more commenting and JSDocs, but otherwise I don't see any issues.

**Review by Justin-Sherfey — APPROVED** [review]

**Review by LutherNa — CHANGES_REQUESTED** [review]
This request cannot be merged until the corresponding merge to BackendChatroom goes through. Once that's done it's golden.

**Review by LutherNa — APPROVED** [review]

**Pull request merged**